.vscode/launch.json

```diff
@@ -5,6 +5,29 @@
     "version": "0.2.0",
     "configurations": [
         {
+            "type": "PowerShell",
+            "request": "launch",
+            "name": "Create TunnelTestEnv Servers",
+            "script": "${file}",
+            "args": ["-SprintSignoff -SubscriptionId 34f2279c-2de9-44d7-98c6-e0533cd70bd8 -VmName myVMName"],
```

Comment:
Subscription ID needs to be backed of all the configurations out before this is merged in. We may want to look at something like this that doesn't get checked in. https://code.visualstudio.com/docs/editor/variables-reference#_input-variables

Reply:
For now, backing it out may be the simplest method of moving forward, the above is a suggestion for how we may want to do this in the future.